.github/workflows/auto-updater.yaml

```diff
@@ -16,10 +16,11 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt -y install xvfb libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xkb1 libxcb-shape0 libxkbcommon-x11-0
-          wget -nv https://dlsrc.getmonero.org/gui/monero-gui-linux-x64-v0.18.0.0.tar.bz2
-          tar -xf monero-gui-linux-x64-v0.18.0.0.tar.bz2
-          mv monero-gui-v0.18.0.0/monero-wallet-gui .
-          rm -rf monero-gui-linux-x64-v0.18.0.0.tar.bz2 monero-gui-v0.18.0.0
+          previous_tag=$(curl "https://api.github.com/repos/monero-project/monero-gui/tags" | jq -r '.[1].name')
```

Comment:
Totally inconsequential since this is just a workflow, but i prefer to do curl -s